src/styles/base.scss

```
@@ -0,0 +1,185 @@
+}
+
+button {
+	user-select: none;
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+	padding: 8px 10px;
+	gap: 4px;
+
+	background-color: var(--surface);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)